tokio/src/future/mod.rs

```diff
@@ -6,6 +6,7 @@
 pub(crate) mod maybe_done;
 
 mod poll_fn;
+#[allow(unused_imports)]
 pub use poll_fn::poll_fn;
```

Comment:
Does this work?
```suggestion
pub(crate) use poll_fn::poll_fn;
```

Reply:
@Darksonn 
Hmmm, that change doesn't seem to work. 🤔 (I tried it on the [fork branch](https://github.com/mox692/tokio/pull/4/commits/816ebb7e257d3a7df5d10bc43af01135b7fca2ec).)

```
error[E0364]: `poll_fn` is only public within the crate, and cannot be re-exported outside
 --> tokio/src/macros/support.rs:2:13
  |
2 |     pub use crate::future::poll_fn;
  |             ^^^^^^^^^^^^^^^^^^^^^^
  |
note: consider marking `poll_fn` as `pub` in the imported module
 --> tokio/src/macros/support.rs:2:13
  |
2 |     pub use crate::future::poll_fn;
  |             ^^^^^^^^^^^^^^^^^^^^^^

error: unused import: `crate::future::poll_fn`
 --> tokio/src/macros/support.rs:2:13
  |
2 |     pub use crate::future::poll_fn;
  |             ^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `-D unused-imports` implied by `-D warnings`
  = help: to override `-D warnings` add `#[allow(unused_imports)]`

```

I've tried other things, but I haven't figured out a good way.